src/components/document-list/_macro.spec.js

```diff
@@ -4,472 +4,359 @@ import * as cheerio from 'cheerio';
                 }),
             );
-
-            expect($('#some-id').length).toBe(1);
+            test('THEN: passes jest-axe checks', async () => {
+                const results = await axe($.html());
+                expect(results).toHaveNoViolations();
+            });
+
+            test('THEN: outputs three document items', () => {
+                expect($('.ons-document-list__item').length).toBe(3);
+            });
+            test('THEN: has expected url for the title', () => {
+                expect($('.ons-document-list__item-title a').attr('href')).toBe('#0');
+            });
         });
-
-        it('has custom classes applied', () => {
-            const $ = cheerio.load(
-                renderComponent('document-list', {
-                    classes: 'custom-class',
-                    documents: [EXAMPLE_DOCUMENT_LIST_BASIC, EXAMPLE_DOCUMENT_LIST_BASIC, EXAMPLE_DOCUMENT_LIST_BASIC],
-                }),
-            );
-
-            expect($('.ons-document-list').hasClass('custom-class')).toBe(true);
+    });
+    describe('GIVEN: Params: description', () => {
+        describe('WHEN: description is provided', () => {
+            test('THEN: has expected description', () => {
+                const $ = cheerio.load(renderComponent('document-list', { documents: [EXAMPLE_DOCUMENT_LIST_BASIC] }));
+                const title = $('.ons-document-list__item-description').html().trim();
+                expect(title).toBe('Some description');
+            });
         });
+    });
 
-        it('outputs the correct number of document items', () => {
-            const $ = cheerio.load(
-                renderComponent('document-list', {
-                    documents: [EXAMPLE_DOCUMENT_LIST_BASIC, EXAMPLE_DOCUMENT_LIST_BASIC, EXAMPLE_DOCUMENT_LIST_BASIC],
-                }),
-            );
-
-            expect($('.ons-document-list__item').length).toBe(3);
+    describe('GIVEN: Params: id', () => {
+        describe('WHEN: id is provied', () => {
+            test('THEN: has the provided id attribute', () => {
+                const $ = cheerio.load(
+                    renderComponent('document-list', {
+                        id: 'some-id',
+                        documents: [EXAMPLE_DOCUMENT_LIST_BASIC, EXAMPLE_DOCUMENT_LIST_BASIC, EXAMPLE_DOCUMENT_LIST_BASIC],
+                    }),
+                );
+                expect($('#some-id').length).toBe(1);
+            });
         });
+    });
 
-        it('has the correct container if `fullWidth`', () => {
-            const $ = cheerio.load(
-                renderComponent('document-list', {
-                    documents: [{ ...EXAMPLE_DOCUMENT_LIST_BASIC, featured: true, fullWidth: true }],
-                }),
-            );
-
-            expect($('.ons-container').length).toBe(1);
+    describe('GIVEN: Params: custom-class', () => {
```

Comment:
```suggestion
    describe('GIVEN: Params: classes', () => {
```
"classes" is the param name